internal/core/autocomplete_test.go

```diff
@@ -125,8 +130,13 @@ func TestAutocomplete(t *testing.T) {
 	t.Run("scw test flower create leaves.0.", run(&testCase{Suggestions: AutocompleteSuggestions{"leaves.0.size="}}))
 	t.Run("scw test flower create leaves.0.size=M leaves", run(&testCase{Suggestions: AutocompleteSuggestions{"leaves.1.size="}}))
 	t.Run("scw test flower create leaves.0.size=M leaves leaves.1.size=M", run(&testCase{WordToCompleteIndex: 5, Suggestions: AutocompleteSuggestions{"leaves.2.size="}}))
-	t.Run("scw test flower delete f", run(&testCase{Suggestions: AutocompleteSuggestions{"flower="}}))
-	t.Run("scw test flower delete flower f", run(&testCase{Suggestions: nil}))
+	t.Run("scw test flower delete ", run(&testCase{Suggestions: AutocompleteSuggestions{"anemone", "hibiscus"}}))
+	t.Run("scw test flower delete w", run(&testCase{Suggestions: nil}))
```

Comment:
why are there no suggestions if there are for `scw test flower delete hibiscus w` ?